Resources/Prototypes/Entities/Objects/Weapons/Melee/fireaxe.yml

```
@@ -86,6 +89,7 @@
     quickEquip: false
     slots:
     - back
+    - suitStorage
```

```suggestion
    - suitStorage  # Goobstation
```
Don't forget to tag your changes as Goobstation changes.